README.md

```diff
@@ -1 +1,44 @@
-# Speak-Fluently
\ No newline at end of file
+# Speak-Fluently
+
+## Set up an Online Education Website
+
+In this project, we will build an online education website offering online English lessons to students from all over the world.
+
+#### Project requirements 
+
```

Comment:
- [ ] [OPTIONAL]: To make this README file even more professional, you should consider adding a screenshot of your project.